ext/curl/interface.c

```diff
@@ -908,6 +907,29 @@ PHP_MINIT_FUNCTION(curl)
 	REGISTER_CURL_CONSTANT(CURL_VERSION_SPNEGO);
 	REGISTER_CURL_CONSTANT(CURL_VERSION_SSL);
 	REGISTER_CURL_CONSTANT(CURL_VERSION_SSPI);
+	REGISTER_CURL_CONSTANT(CURLVERSION_NOW);
```

Comment:
I don't think the CURLVERSION_* constants should be exposed. They are needed to properly interpret curl_version_info() output in the implementation, but they aren't useful to end users.

Reply:
IIUC, why is required to pass the curl version as argument 1 to `curl_version()` (which by default get `CURLVERSION_NOW` as value).

Reply:
I've removed the changes related to `CURLVERSION_` constants in order to get this PR merged. IMO, the requirement for these constants could be discussed in a separate PR.

Reply:
It seems to me that calling `curl_version()` with anything else but the default (`CURLVERSION_NOW`) is bad, and that PHP should not even allow to pass anything else.

Reply:
Agreed. We should drop that parameter.